.bit/responses/5.5-Week 5 Step 5.md

```diff
@@ -0,0 +1,29 @@
+---
+files: n/a
+stepType: comment
+scripts: n/a
+week: 5
+step: 5
+name: Week 5 Step 5
+---
+Week 5 Step 5 ◯◯◯◯⬤◯ | 🕐 Estimated completion: 1-5 hours
+
+## Start Coding!
+
+### ✅  Task:
+
+- [ ] Based on your schedule, begin your first task
```

Comment:
Depending on whether we choose to have them put their project in a second repo or not, these tasks will change